client/signup/steps/site-type/index.jsx

```diff
@@ -94,7 +112,7 @@ class SiteType extends Component {
 
 		const headerText = translate( 'What kind of site are you building?' );
 		const subHeaderText = translate(
-			'This is just a starting point. You can add or change features later.'
+			"Choose a site type, answer a few questions, and we'll start you off with a fresh site and customized content."
```

Comment:
@ranh This okay in your view, or should we change the copy in another PR?

Reply:
I've reverted it so we can look at it another day. It's not vital for this AB test to move forward.